### ISSUE TITLE
Error while connecting print using USB in windows 

### ISSUE THREAD

**Issue by RachanaTri** [mike42/escpos-php#1067]
Hello
I am developing laravel based website and used this package for printing receipt.
I have connected XP-80 printer using USB, but it is giving me error : 
**copy(\\computer\XP-80): failed to open stream: No such file or directory**

Here is my code ;
use Mike42\Escpos\Printer;
use Mike42\Escpos\PrintConnectors\WindowsPrintConnector;
$connector = new WindowsPrintConnector("smb://computer/XP-80");
$printer = new Printer($connector);
$printer -> text("Hello World!\n");
$printer -> cut();
$printer -> close();

Please suggest what I am missing.
Thank You

**Comment by Muraveiko** [comment]
your website is hosted on the local network of the organization and the printer is connected to the USB port on the server?

**Comment by RachanaTri** [comment]
No, website is on localhost and printer is attached using USB with laptop.

**Comment by rochelo** [comment]
Encontraron la solucion??
